tools/converter/reports_converter.cpp

```diff
@@ -0,0 +1,104 @@
+#include <cstring>
+#include <fstream>
+#include <iostream>
+#include <map>
+#include <vector>
+
+#include "binary_reader/ReadManager.h"
+#include <bbp/sonata/reports.h>
+#include <utils/logger.h>
+
+using namespace bbpReader;
+
+int get_steps_to_write (const char* var_name) {
+    int steps_to_write = 10;
+    char* var_value = getenv(var_name);
+    if (var_value != nullptr && atoi(var_value) > 0) {
+            steps_to_write = atoi(var_value);
+    }
+    return steps_to_write;
+}
+
+int main(int argc, char* argv[]) {
+    if (argc != 3) {
+        logger->error("Wrong number of arguments.");
+        logger->info("Try: ./reports_converter <filename> <[--soma, --compartment]>");
+        logger->info("Example: ./reports_converter soma.bbp --soma");
+        return 0;
+    }
+    const char* file_name = argv[1];
+    std::string report_type = argv[2];
+    logger->info("Trying to convert '{}' binary report...'", file_name);
+    std::ifstream f(file_name);
+    if (!f.good()) {
+        logger->error("File '{}' not found!", file_name);
+        return 0;
+    }
+
```

Comment:
`nitpick / suggestion`: it would have been nice if `reports_converter --help` or `reports_converter -h` would have given some useful info about the usage.

Reply:
With the same info that when you dont pass the number of arguments i guess?
```
"Try: ./reports_converter <filename> <[--soma, --compartment]>"
"Example: ./reports_converter soma.bbp --soma"
```

Reply:
(As this is simple tool, I didn't want to suggest [CLI11](https://github.com/CLIUtils/CLI11))

I meant, if I see some new tool, I typically do -h or --help to see what I can pass.

Reply:
I thought it was overkill to use CLI11